azext_iot/tests/central/__init__.py

```diff
@@ -6,27 +6,53 @@
 
 import os
 import time
-
+import pytest
+from typing import Tuple
 from azext_iot.tests import CaptureOutputLiveScenarioTest
 from azext_iot.tests.conftest import get_context_path
 from azext_iot.common import utility
-from azext_iot.central.models.enum import Role
+from azext_iot.central.models.enum import Role, UserTypePreview, UserTypeV1, ApiVersion
 
 APP_ID = os.environ.get("azext_iot_central_app_id")
 APP_PRIMARY_KEY = os.environ.get("azext_iot_central_primarykey")
 APP_SCOPE_ID = os.environ.get("azext_iot_central_scope_id")
 DEVICE_ID = os.environ.get("azext_iot_central_device_id")
 TOKEN = os.environ.get("azext_iot_central_token")
 DNS_SUFFIX = os.environ.get("azext_iot_central_dns_suffix")
+STORAGE_CSTRING = os.environ.get("azext_iot_central_storage_cstring")
```

Comment:
We need to add new env vars for testing to `pytest.ini.example`

Reply:
As well as CI definitions/scripts.

Reply:
done